metron-interface/metron-alerts/src/app/service/cluster-metadata.service.ts

```diff
@@ -16,14 +16,20 @@
  * limitations under the License.
  */
 import {Injectable} from '@angular/core';
+import {Headers, RequestOptions} from '@angular/http';
 import {Observable} from 'rxjs/Rx';
 import {Http} from '@angular/http';
+
+import {INDEXES} from '../utils/constants';
+import {MetronRestApiUtils} from '../utils/metron-rest-api-utils';
 import {ColumnMetadata} from '../model/column-metadata';
 import {DataSource} from './data-source';
+import {HttpUtil} from '../utils/httpUtil';
 
 @Injectable()
 export class ClusterMetaDataService {
```

Comment:
Shouldn't this be in a search.service.ts class instead of having it's own service class?

Reply:
Done